client/src/main/resources/java9_invisible_packages.properties

```diff
@@ -0,0 +1 @@
+sun.java2d=sun.java2d
```

Comment:
Why is this a properties file? Wouldn't a plain text file be sufficient? (Like e.g. client/src/main/resources//jdkPureMethods.txt)

Reply:
I guess it makes it easier to read using ResourceBundle.

Reply:
But it's a bit weird, you need to specify every package twice as key=value with key being the same as value.

Reply:
Refactored to use java9_invisible_packages.txt